cylc/rose/utilities.py

```diff
@@ -454,9 +444,7 @@ def get_cli_opts_node(opts=None, srcdir=None):
         )
 
     # Specialised treatement of optional configs.
-    if 'opts' not in newconfig:
-        newconfig['opts'] = ConfigNode()
-        newconfig['opts'].value = ''
+    newconfig['opts'].value = ''
```

Comment:
Unnecessary?
```suggestion
```

Reply:
No. Try commenting it out and see what happens to the tests: The  `merge_opts` function requires the `newconfig['opts']` object to have something assigned to the value attribute.